src/hipscat_import/catalog/file_readers.py

```diff
@@ -126,6 +126,7 @@ def __init__(
         column_names=None,
         type_map=None,
         separator=",",
+        parquet_kwargs=None,
```

Comment:
Could you please add "Parameters" to the class-level docstring to explain the difference between `kwargs` and `parquet_kwargs`

Reply:
Thank you for fixing it!